nengo_gui/components/action.py

```diff
@@ -64,7 +64,7 @@ def __init__(self, viz_sim, component, new_cfg, old_cfg):
 
     def load(self, cfg):
         for k, v in iteritems(cfg):
-            setattr(self.viz_sim.viz.config[self.component.template], k, v)
+            setattr(self.viz_sim.config[self.component.template], k, v)
```

Comment:
Ah, that should be ```sim```....    Hmm, and scrolling down a bit it looks like I completely didn't update the RemoveGraph undo function...  oops....